bundles/org.openhab.binding.bondhome/src/main/java/org/openhab/binding/bondhome/internal/handler/BondDeviceHandler.java

```diff
@@ -619,6 +642,7 @@ public synchronized void updateChannelsFromState(@Nullable BondDeviceState updat
             fanOn = updateState.fpfanPower != 0;
             updateState(CHANNEL_FAN_POWER, fanOn ? OnOffType.OFF : OnOffType.ON);
             updateState(CHANNEL_FAN_SPEED, new PercentType(updateState.fpfanSpeed));
+            updateState(CHANNEL_RAW_FAN_SPEED, new DecimalType(updateState.fpfanSpeed));
```

Comment:
this seems buggy. fpfanspeed should either be 1-100, or 1-{max speed}, but not both. while we're at it, is fpfanspeed 0 when the fan is off?

Reply:
ohhh fpfanspeed is _fireplace_ fan speed. your handleCommand implementation definitely does not handle a raw fan speed with a fireplace fan

Reply:
I'll look deeper at this... for some reason I thought this would be ok, because speed is speed if you feed it, but  maybe i missed the winkle with the fireplace fan.  Seems like an easy fix via an if statement.

Reply:
after talking to @ccutrer in another medium, we thought we'd just not set this for fireplace fans for now